### ISSUE TITLE
Make adding captions easy from the table

### ISSUE THREAD

**Issue by nikkimk** [ckeditor/ckeditor5#5577]
## 📝 Provide a description of the new feature
A11y feature: As part of the floating table menu, there should be toggle button to add a table caption.
(Otherwise users my try to fake it inaccessibly with merged cells.)

*What is the expected behavior of the proposed feature?*
If a button is toggled, a place for the table caption appears.

---

If you'd like to see this feature implemented, add a 👍 reaction to this post.

**Comment by Mgsy** [comment]
Hello, we have an open feature request for table captions - https://github.com/ckeditor/ckeditor5/issues/3204. Feel free to add 👍 under the first post to bump its popularity.
